source/manual/concourse.html.md

```diff
@@ -15,22 +15,48 @@ The Reliability Engineering team runs a [hosted Concourse](https://cd.gds-reliab
 
-> **Note**
->
-> To log in to Concourse, you need to be a member of the GOV.UK Production GitHub team
-
 ## Teams
 
 GOV.UK currently has one team named [govuk-tools](https://cd.gds-reliability.engineering/teams/govuk-tools).
 
+To view pipelines within this Concourse team, you need to be a member of the GOV.UK GitHub team.
+To change pipelines within this Concourse team, you need to be a member of the GOV.UK Production GitHub team.
+
 ## Pipelines
 
-GOV.UK currently has two pipelines named [info](https://cd.gds-reliability.engineering/teams/govuk-tools/pipelines/info) and [operations](https://cd.gds-reliability.engineering/teams/govuk-tools/pipelines/operations).
+### What pipelines do we currently have?
+
+#### The coronavirus forms pipelines
 
-### The operations pipeline
+More information about these can be found [on the docs page for the forms](/manual/covid-19-services.html#deployment).
+
+#### The operations pipeline
 
 We currently use the `operations` pipeline to [mirror all GOV.UK GitHub repositories to AWS CodeCommit](repository-mirroring.html).
 
-### The info pipeline
+#### The info pipeline
 
 The `info` pipeline is a meta pipeline. Its main use is as a method to store secrets that can then be used in other pipelines. For example, the repository mirroring job uses GitHub and AWS credentials that are stored as secrets using this method. How to add and remove secrets is documented [in the Reliability Engineering manual](https://reliability-engineering.cloudapps.digital/continuous-deployment.html#secrets).
+
+### Creating new pipelines
+
+The above pipelines serve as good examples for how to do continuous deployment and scheduled tasks with Concourse.
+
+Whenever a pipeline YAML is created or changed, it needs to be applied for the changes to take effect.
+
+Some of our Concourse pipelines use the beta Concourse "self-update" feature that runs a task whenever a change to the pipeline YAML is merged to master. For others, like the [repo mirroring job](/manual/repository-mirroring.html), or if you're writing a new Concourse pipeline, you have to run `fly set-pipeline` manually:
+
+1. If this is your first time using Concourse, download the `fly` CLI by clicking the appropriate OS logo at the bottom right corner of the [team page](https://cd.gds-reliability.engineering/teams/govuk-tools) and move it to somewhere in your `$PATH`. If that doesn't download an executable file, try `brew cask install fly`.
+1. Set a target for the team you want to login to: `fly login -c https://cd.gds-reliability.engineering -n govuk-tools -t cd-govuk-tools`
+1. Navigate to the folder that contains the pipeline config YAML file
+1. Run `fly -t cd-govuk-tools set-pipeline -p [pipeline name] -c [pipeline config YAML file name]`
+1. Visit the provided URL to log in
+1. Double-check the diff to ensure you're happy with what is about to be applied
+1. Enter `y` to make the changes
```

Comment:
These detailed steps might not be necessary, since they might change, and `fly` gives prompts.